spec/post_spec.rb

```diff
@@ -0,0 +1,69 @@
+require_relative 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject do
+    Post.new(title: 'first_post', text: 'This is my first post', comments_counter: 1, likes_counter: 1, author_id: 1)
+  end
```

Comment:
- [ ] Dear @gealsanchez, according to the primary requirements for the project, any custom methods in the models are expected to have a corresponding unit test. As part of that, we need to add a test for the `update_posts_counter` method inside your `Post` model.